### PR TITLE
Fix policy testing for tv5

### PR DESCRIPTION
**Pull request by jadechoghari**
## Title
Fix policy testing for Transformers v5

Torch 2.10 introduces slight differences in logits compared to Torch 2.7.1. However, these differences do not affect final performance in any way. In fact, the mean and standard deviation of the action output distribution remain identical to Torch 2.7.1, as verified through internal simulation benchmarks.

We also skip the CI test for Wall-X, as it leads to out-of-memory (OOM) errors in CI environments.

### CODE REVIEW

**Comment by imstevenpmwork** [comment]
Full CI run: https://github.com/huggingface/lerobot/actions/runs/22455322229
